src/content/docs/es/plugin/notification.mdx

```diff
@@ -0,0 +1,319 @@
+---
+title: Notificaciones
+description: Envía notificaciones nativas al usuario.
+i18nReady: true
+plugin: notification
+---
+
+import PluginLinks from '@components/PluginLinks.astro';
+import Compatibility from '@components/plugins/Compatibility.astro';
+import PluginPermissions from '@components/PluginPermissions.astro';
+import { Tabs, TabItem, Steps } from '@astrojs/starlight/components';
+import CommandTabs from '@components/CommandTabs.astro';
+
+<PluginLinks plugin={frontmatter.plugin} />
+
+Envía notificaciones nativas al usuario utilizando el plugin de notificaciones.
+
+## Plataformas soportadas
+
+<Compatibility plugin={frontmatter.plugin} />
+
+## Configuración
+
+Instala el plugin de notificaciones para comenzar.
+
+<Tabs>
+    <TabItem label="Automatic">
+
+        Utiliza el gestor de paquetes de tu proyecto para añadir la dependencia:
+
+        <CommandTabs npm="npm run tauri add notification"
+            yarn="yarn run tauri add notification"
+            pnpm="pnpm tauri add notification"
+            bun="bun tauri add notification"
+        deno="deno task tauri add notification"
+            cargo="cargo tauri add notification" />
+
+
+    </TabItem>
+    <TabItem label="Manual">
+        <Steps>
+
+        1. Ejecuta el siguiente comando en el directorio `src-tauri` para añadir el plugin a las dependencias del proyecto en `Cargo.toml`:
+
+           ```sh frame=none
+           cargo add tauri-plugin-notification
+           ```
+
+        2. Modifica `lib.rs` para inicializar el plugin:
+
+           ```rust title="src-tauri/src/lib.rs" ins={4}
+           #[cfg_attr(mobile, tauri::mobile_entry_point)]
+           pub fn run() {
+               tauri::Builder::default()
+                   .plugin(tauri_plugin_notification::init())
+                   .run(tauri::generate_context!())
+                   .expect("error while running tauri application");
+           }
+           ```
+
+        3. Si deseas utilizar notificaciones desde JavaScript, instala también el paquete npm:
+
+
+            <CommandTabs
+                npm="npm install @tauri-apps/plugin-notification"
+                yarn="yarn add @tauri-apps/plugin-notification"
+                pnpm="pnpm add @tauri-apps/plugin-notification"
+                deno="bun add npm:@tauri-apps/plugin-notification"
+                bun="bun add @tauri-apps/plugin-notification"
+            />
+
+        </Steps>
+    </TabItem>
+
+</Tabs>
+
+## Uso
+
+Aquí hay algunos ejemplos de cómo usar el plugin de notificaciones:
+
+- [Enviar notificaciones a los usuarios](#enviar-notificación)
```

Comment:
We should make sure diacritics work after "anchor slugs" are created.

Reply:
I checked that. There's a preview deployment automatically generated and you can see that it works:

https://deploy-preview-3137--tauri-v2.netlify.app/es/plugin/notification/#enviar-notificaci%C3%B3n